packages/p2p/src/stores/floating-rate-store.js

```diff
@@ -0,0 +1,80 @@
+import { action, computed, observable } from 'mobx';
```

Comment:
Wasn't this store added in another PR? Why are the same changes here since that one still hasn't been merged and still has review issues?

Reply:
I needed to make use of the store for this feature, hence it is copied here. The changes I make in that will be merged here as well

Reply:
Just add what you need for this change, if there are issues in that one, you would have to re-fix them all. You can fix any conflicts when you need to.

Reply:
Will remove the additional functions and properties